assemblies/static/src/main/resources/LICENSE

```diff
@@ -1858,73 +1858,73 @@ This product bundles threetenbp 1.5.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-html-module 2.1.0, which is available under a
+This product bundles tika-parser-html-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-image-module 2.1.0, which is available under a
+This product bundles tika-parser-image-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-mail-commons 2.1.0, which is available under a
+This product bundles tika-parser-mail-commons 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-mail-module 2.1.0, which is available under a
+This product bundles tika-parser-mail-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-microsoft-module 2.1.0, which is available under a
+This product bundles tika-parser-microsoft-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-miscoffice-module 2.1.0, which is available under a
+This product bundles tika-parser-miscoffice-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-news-module 2.1.0, which is available under a
+This product bundles tika-parser-news-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-ocr-module 2.1.0, which is available under a
+This product bundles tika-parser-ocr-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-pdf-module 2.1.0, which is available under a
+This product bundles tika-parser-pdf-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-pkg-module 2.1.0, which is available under a
+This product bundles tika-parser-pkg-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-text-module 2.1.0, which is available under a
+This product bundles tika-parser-text-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-xml-module 2.1.0, which is available under a
+This product bundles tika-parser-xml-module 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-xmp-commons 2.1.0, which is available under a
+This product bundles tika-parser-xmp-commons 2.3.0, which is available under a
 "Apache 2.0" license. For details, see LICENSE.
 
-This product bundles tika-parser-zip-commons 2.1.0, which is available under a
+This product bundles tika-parser-zip-commons 2.3.0, which is available under a
```

Comment:
@gvdutra all the poi 4.1.2 refs in license file can go now because those were transitive deps of tika - tika now uses poi 5.2.0 - which matches what hop excel components use

Reply:
Hello, @pjfanning thanks so much, so that being said, I will be removing all of them.

Also, sorry for taking this ticket, I did the change and committed and then I realized that you probably were working on it.

Reply:
@gvdutra I didn't do any work about this issue, I just noticed the PR and thought the license file needed to be maintained manually - thanks for taking it on